datasets/mc_TT_inclusive.json

```diff
@@ -1,5 +1,5 @@
 {
-  "/TT_TuneCUETP8M1_13TeV-powheg-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12-v1/MINIAODSIM": {
+  "/TT_TuneCUETP8M1_13TeV-powheg-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12_ext3-v1/MINIAODSIM": {
```

Comment:
actually in some instances there are both the 'ext' and 'not-ext' samples available, is this the case for this one ?